src/parsing/medium.py

```diff
@@ -347,12 +347,16 @@ async def type_fallback(self) -> bool:
             # create type fallback medium
             self.type_fallback_medium = self.typeFallbackTo(fallback_urls)
             if await self.type_fallback_medium.validate():
-                logger.debug(f"Medium {self.original_urls[0]}"
-                             + (f' ({self.info})' if self.info else '')
-                             + f" type fallback to '{self.type_fallback_medium.type}'"
-                             + (f'({self.type_fallback_medium.original_urls[0]})'
-                                if not self.typeFallbackAllowSelfUrls
-                                else ''))
+                logger.debug(
+                            f"Medium {self.original_urls[0]}"
+                            + (f' ({self.info})' if self.info else '')
+                            + f" type fallback to '{self.type_fallback_medium.type}'"
+                            + (
+                                ''
+                                if self.typeFallbackAllowSelfUrls
+                                else f'({self.type_fallback_medium.original_urls[0]})'
+                            )
```

Comment:
```suggestion
                    f"Medium {self.original_urls[0]}"
                    + (f' ({self.info})' if self.info else '')
                    + f" type fallback to '{self.type_fallback_medium.type}'"
                    + (
                        ''
                        if self.typeFallbackAllowSelfUrls
                        else f'({self.type_fallback_medium.original_urls[0]})'
                    )
```

Redundant indentation.